scripts/create-update.ps1

```diff
@@ -0,0 +1,92 @@
+<#
+Copyright (c) AppDynamics, Inc., and its affiliates 2015
+All rights reserved
+
+Build script for wpt-driver.
+
+REQUIREMENTS:
+- msbuild.exe and python.exe (required by the Chrome compile.cmd) should be in
+  the path
+
+This script has been tested in GitHub's Git Shell.
+Executing it in a regular PowerShell may also work, but you will have to
+setup the path for each dependencies.
+#>
+
+param(
+    [string]$configuration = "Release"
+)
+
+$wptroot = $PSScriptRoot + "\.."
+
+"*** Building wpt-driver with $configuration configuration"
+
+# ZipFiles implementation from:
+# http://stackoverflow.com/a/13302548
+function ZipFiles( $zipFilename, $sourceDir)
```

Comment:
Is it worth checking the validity of the produced zip or is the process trustworthy enough that it would be unnecessary?